docs/framework/data/adonet/querying-datasets-linq-to-dataset.md

```diff
@@ -4,9 +4,9 @@ ms.date: "03/30/2017"
 ms.assetid: bb68d2e4-623d-4d60-85e3-965254f6fee7
 ---
 # Querying DataSets (LINQ to DataSet)
-After a <xref:System.Data.DataSet> object has been populated with data, you can begin querying it. Formulating queries with LINQ to DataSet is similar to using [!INCLUDE[vbteclinqext](../../../../includes/vbteclinqext-md.md)] against other [!INCLUDE[vbteclinq](../../../../includes/vbteclinq-md.md)]-enabled data sources. Remember, however, that when you use [!INCLUDE[vbteclinq](../../../../includes/vbteclinq-md.md)] queries over a <xref:System.Data.DataSet> object you are querying an enumeration of <xref:System.Data.DataRow> objects, instead of an enumeration of a custom type. This means that you can use any of the members of the <xref:System.Data.DataRow> class in your [!INCLUDE[vbteclinq](../../../../includes/vbteclinq-md.md)] queries. This lets you to create rich, complex queries.  
+After a <xref:System.Data.DataSet> object has been populated with data, you can begin querying it. Formulating queries with LINQ to DataSet is similar to using Language-Integrated Query (LINQ) against other LINQ-enabled data sources. Remember, however, that when you use LINQ queries over a <xref:System.Data.DataSet> object you are querying an enumeration of <xref:System.Data.DataRow> objects, instead of an enumeration of a custom type. This means that you can use any of the members of the <xref:System.Data.DataRow> class in your LINQ queries. This lets you to create rich, complex queries.  
```

Comment:
```suggestion
After a <xref:System.Data.DataSet> object has been populated with data, you can begin querying it. Formulating queries with LINQ to DataSet is similar to using Language-Integrated Query (LINQ) against other LINQ-enabled data sources. Remember, however, that when you use LINQ queries over a <xref:System.Data.DataSet> object, you're querying an enumeration of <xref:System.Data.DataRow> objects instead of an enumeration of a custom type. This means that you can use any of the members of the <xref:System.Data.DataRow> class in your LINQ queries. This lets you create rich, complex queries.  
```